docs/cli.md

```diff
@@ -29,17 +34,303 @@ Common Flags
 | --insecure-skip-tls-verify | Skip SSL certificate validation (will make HTTPS insecure) |
 | --help (-h)                | Display help for specified command |
 
-oc get
------------
+## Basic Commands
+
+### oc
+
+This displays the list of available commands.
+
+### oc login
+
+This retrieves a session token that allows you to act as a user.
+Invoked without args, `oc login` prompts for a username and a password.
+For example, these two invocations are identical:
+
+```bash
+$ oc login
+Username: test
+Password: test
+
+$ oc login -u test -p test
+```
+
+See also [`oc logout`](#oc-logout) and [`oc whoami`](#oc-whoami).
+
+### oc new-project
+
+This creates a new project, with you as the project admin.
```

Comment:
This needs to get updated first in the new-project help message and then back here. I am ok with it for now.